R/model.R

```diff
@@ -329,6 +329,10 @@ naomi_model_frame <- function(area_merged,
   ##           of a Spectrum file and then calibrated. Currently no way to know if areas
   ##           comparise only part of a Spectrum file, so can't address.
 
+  if (!all(mf_areas[["area_id"]] %in% unique(population_agesex$area_id))) {
```

Comment:
If the error message is about the area level selected, why is the conditional here referring to the area IDs (not the area level)?

Reply:
We don't area level available in the population data to easier to check that these intersect and then pull that in if they don't for the error message if needed